src/main/java/org/symphonyoss/integration/webhook/zapier/parser/ZapierNullParser.java

```diff
@@ -46,7 +47,8 @@
    */
   @Override
   public String parse(String eventType, JsonNode payload) throws ZapierParserException {
-    return null;
+    String message = String.format("Event %s not handled by the Zapier parsers", eventType);
+    throw new WebHookUnprocessableEntityException(message);
```

Comment:
We should return 422 on all cases that we get an empty message from the parser... Return null here and handle null/empty messages for all parsers.